docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile

```diff
@@ -48,22 +48,21 @@ Vagrant.configure(2) do |config|
       # binary we have stored that is stable and usable by Ubuntu.
       # DO NOT CHANGE THIS VERSION, unless you explicitly and successfully test a new
       # Docker binary with the Ubuntu vagrant libnetwork demonstration.
-      host.vm.provision :shell, inline: "wget -qO /usr/bin/docker https://github.com/projectcalico/calico-docker/releases/download/v0.5.4/docker", :privileged => true
+      host.vm.provision :shell, inline: "wget -qO /usr/bin/docker https://master.dockerproject.org/linux/amd64/docker", :privileged => true
 
-      # Docker uses Consul for clustering. Install just on the first host.
+      # Calico uses etcd for clustering, and Docker uses it for clustering.
+      # Install it on the first host only.
       if i == 1
-        # Download consul and start.
-        host.vm.provision :shell, inline: <<-SHELL
-          sudo apt-get install -y unzip
-          curl -L --silent https://dl.bintray.com/mitchellh/consul/0.5.2_linux_amd64.zip -o consul.zip
-          unzip consul.zip
-          chmod +x consul
-          nohup ./consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -client #{primary_ip} > consul.log &
-        SHELL
+        host.vm.provision :docker do |d|
```

Comment:
Hmmmm, I'm not sure that we can use a docker version of etcd if Docker is meant to be using it as a cluster store...